LeetCode/0060_Permutation_Sequence.py

```diff
@@ -0,0 +1,21 @@
+class Solution:
+    def getPermutation(self, n: int, k: int) -> str:
+        self.k = k
+    
+        def backtrack(path, remain):
+            if not remain:
+                self.k -= 1
+                if self.k == 0:
+                    return path
+                return ''
+                
+            for i in range(len(remain)):
+                if self.k > math.factorial(len(remain)-1):
+                    self.k -= math.factorial(len(remain)-1)
```

Comment:
```suggestion
                    self.k -= math.factorial(len(remain) - 1)
```

Reply:
`math.factorial(len(remain) - 1)` is counted twice you could assign it to a variable